### ISSUE TITLE
New TSV row should allow users to lookup TAs or TWs

### ISSUE THREAD

**Issue by birchamp** [unfoldingWord/gateway-edit#506]
Just like when we create a new row for Translation Notes in tC Create the new row dialog gE should provide users with a dropdown for looking up the related resource.  

Scenario: User wants to add a row to Translation Notes
Given that the user clicks the plus sign on the Translation notes card
When the new row dialog comes up
Then the support reference field is a dropdown dialog listing all available figure of speech TA articles.

Scenario: User wants to add a row to Translation Word Links
Given that the user clicks the plus sign on the Translation Word Links card
When the new row dialog comes up
Then the TWLink field is a dropdown dialog listing all available figure of speech TW articles.

DoD:
---
Users are able to select from a list of links rather than having to go look up the file.

**Comment by elsylambert** [comment]
@kintsoogi https://deploy-preview-560--gateway-edit.netlify.app/ Not seeing the dropdown for TW articles. It shows as an editable text box. 
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL2k5QVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--b0714c7484a91e79187918d8f08f855eff6c4e22/Screenshot%202023-11-09%20at%209.46.06%E2%80%AFAM.png)

**Comment by elsylambert** [comment]
@kintsoogi  You are right in saying that TCC works the same as in GWE adding new row in TWLs. Will discuss with @birchamp on the DOD for this issue.
